data/networks.ts

```diff
@@ -111,6 +130,18 @@ const publicChains: ZkSyncNetwork[] = [
       isTestnet: true,
     },
     l1Network: l1Networks.sepolia,
+    settlementChains: [
+      {
+        explorerUrl: "https://sepolia.etherscan.io",
+        name: "Ethereum",
```

Comment:
Just a minor thing. Should we label these so they're more clear that they're Sepolia/Testnet?